src/lib/apiService.ts

```diff
@@ -152,7 +182,8 @@ const getAccountTransactions = async (accountId: string, startDate?: Moment, end
     if (response && response.status === 200 && response.data.data) {
```

Comment:
Looks like these lines have to also be removed to make sure your code is called.